docs/adguard-vpn-for-ios/solving-problems/automation.md

```diff
@@ -12,34 +12,39 @@ But, as it often happens, due to a number of technical nuances, it is impossible
 
 ## Setting up AdGuard VPN automatic activation
 
-Let's assume that on your phone or tablet AdGuard VPN works in *Selective mode*, i.e. only on sites listed in *Exclusions*. In this case, apps’ traffic doesn't get into the VPN tunnel. But you want to use, for example, Twitter with VPN enabled. To avoid turning AdGuard VPN on/off manually every time you interact with the social network, we suggest setting up automation on your iOS device. 
+If you need a VPN for one or more apps, set up AdGuard VPN on your iOS device to automatically turn on and off for them. Go to the Exclusions tab, select General mode and follow the instructions. Here we describe how to create automation for Twitter, but you can choose any other app. 
```

Comment:
set up AdGuard VPN to automatically turn on and off when you open and close them